test/recipes/90-test_rpk.t

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env perl
+# Copyright 2016-2022 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
nit : just `Copyright 2023` as you are adding this file

Reply:
Yeah, I've been working on this for a while. :)